alphagov/paas-rubbernecker
# Prettify things

## What

I've updated some dependencies. Mostly because of vulnerabilities but also due to us running bootstrap framework in beta.

I'm moving some of the stickers around to the left side of the card, meaning we can group days along with the teams and team leads, leaving the left side only for iconic stickers.

I'm restyling the buttons a little, to make them fit a bit more with the rest of the rubbernecker.

## How to review

- Run application `make dependencies build && ./bin/rubbernecker`
- Notice the button changes
- Notice the sticker/label changes
